e2e/paths/caseSubmitted_test.js

```diff
@@ -26,3 +29,14 @@ Scenario('Cannot submit a case unless consent is given', I => {
   I.click('Continue');
   I.seeInCurrentUrl('/submitApplication');
 });
+
+//test skipped for now as after number of tests go on to next page to un-skip remove x from before scenario, clear volumes and run test
+xScenario('Can submit a case and see date submitted', (I, caseViewPage, caseListPage, submitApplicationPage) => {
+  submitApplicationPage.giveConsent();
+  I.continueAndSubmit();
+  I.seeEventSubmissionConfirmation(config.applicationActions.submitCase);
+  caseViewPage.goToCaseList();
+  caseListPage.changeStateFilter('Submitted');
+  let row = caseListPage.findCase(caseId);
+  I.seeSubmissionDate(row);
```

Comment:
I would just do I.seeSubmissionDate(caseListPage.findCase(caseId)) here. Saves the let row ... line

Reply:
👍